README.rst

```diff
@@ -63,8 +63,8 @@ Development
 -----------
 
 For development, first install dependencies. The ones needed for running the
-server are listed in ``requirements.txt``, for running testsuite in
-``requirements-test.txt``. You can install them using your distribution (the
+server are listed in "requirements.txt", for running testsuite in
```

Comment:
Since this is an RST document (which is rendered similar to Markdown), the backticks/backquotes indicate an [inline literal](https://docutils.sourceforge.io/docs/user/rst/quickref.html#inline-markup) which I think fits our standard better than the double-quotes here. We tend to indicate filenames as inline literal text throughout the project; do you dislike that style?